### ISSUE TITLE
Borg Floor Tile Module Cannot Pick up Loose Floor Tiles

### ISSUE THREAD

**Issue by Fordoxia** [ParadiseSS13/Paradise#26751]
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1633

### Issue Description

Title.

### What did you expect to happen?

To be able to pick up loose floor tiles to replenish the stockpile.

### What happened instead?

No interaction.

### Why is this bad/What are the consequences?

Cannot craft floor tiles from metal stockpile to restock floor tile stockpile.

### Steps to reproduce the issue.

Empty out floor tiles. Try to pick up floor tiles from floor.

### When did the problem start happening?

Noticed today.

### Extra information

All other borg stack items still pick up loose stuff, metal, rods, etc.

**Comment by 1080pCat** [comment]
relevant runtime

```dm
[2024-09-11T02:44:50] Runtime in code/game/objects/items/stacks/stack.dm:343: Stack merge attempted on qdeleted target stack.
   proc name: merge without del (/obj/item/stack/proc/merge_without_del)
   usr: CRUNCH ([CKEY]) (/mob/living/silicon/robot)
   usr.loc: The plating (189,69,2) (/turf/simulated/floor/plating)
   src: the floor tiles (/obj/item/stack/tile/plasteel)
   src.loc: the plating (189,69,2) (/turf/simulated/floor/plating)
   call stack:
   the floor tiles (/obj/item/stack/tile/plasteel): merge_without_del
   the floor tiles (/obj/item/stack/tile/plasteel): merge
   ImmediateInvokeAsync(the floor tiles (/obj/item/stack/tile/plasteel), "merge", the floor tiles (/obj/item/stack/tile/plasteel))
   the floor tiles (/obj/item/stack/tile/plasteel): Crossed
   the floor tiles (/obj/item/stack/tile/plasteel): forceMove
   CRUNCH (/mob/living/silicon/robot): put_in_hands
   /datum/stack_recipe (/datum/stack_recipe): post_build
   the metal (/obj/item/stack/sheet/metal/cyborg): ui_act
   /datum/tgui (/datum/tgui): on_message
   /datum/tgui_window (/datum/tgui_window): on_message
   tgui Topic(/list (/list))
   [CKEY] (/client): Topic
   ```

**Comment by Fordoxia** [comment]
Issue is fixed by #26791.
